lib/Elevate/PkgMgr/YUM.pm

```diff
@@ -0,0 +1,351 @@
+package Elevate::PkgMgr::YUM;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::PkgMgr::YUM
+
+Logic wrapping the RHEL based package managers
+
+=cut
+
+use cPstrict;
+
+use File::Copy ();
+
+use Cpanel::OS ();
+
```

Comment:
why the spacing between use statements? this confuses tidy.

Reply:
Habit.  I usually split the use statement into 3 separate groups:

1. CPAN namespace
2. cPanel namespace
3. Elevate namespace